### PR TITLE
init community guide & project/chair list

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: reference/chair.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Coordinate the agenda for joint leadership meetings with the board
+- Send weekly update emails to openv2x-discuss
+
+## Each Cycle
+
+- Manage the initiative tracker
+  - Prepare release tracker eventpad (During PTG), Example:
+  - Update the eventpad link in the wiki
+- Check on the active goals and, if there is no active goal, then ensure that 2 TSC members are signed up to manage the goal selection process.
+- Ensure PTI and testing runtime update is done.
```
Comment (likai99c) [inline reference/chair.md]: PTL?

File: reference/chair.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- After milestone-1, ensure the next release naming is started.
+- During R-9 week, check "PTL and TSC elections".
```
Comment (likai99c) [inline reference/chair.md]: what is R-9?

File: reference/chair.md (hunk 1 above)
Comment (ghost) [reply reference/chair.md]: Project Testing Interface

File: reference/tsc-guide.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Attend the Technical Steering Committee meetings (assuming your timezone allows). **The TSC has meetings every Thursday**; check Eavesdrop. To hold the meeting we must meet quorum, so it's important that you attend as many as possible. If you have topics that should be discussed at the meeting, they should be sent to the chair or vice-chair to be included in the agenda. The agenda should also be publicized to online-document platform to encourage community involvement.
+
+## Governance Core Duties
+
+Book time on your calendar for reviewing open patches in the TSC Repos. As a TSC member, you are a primary maintainer (a core member in  OpenV2X terms) of all the TSC repos. Even if you have already done a review of the patch, there might have been other progress that you should familiarize yourself with and stay up to date on.
+
+## TSC Interactions with Other Governing Bodies
+
+Its encouraged that you attend Board of Directors meetings wherever possible. There are a variety of topics that often come up that the TSC should be weighing in on. Sometimes there are face to face meetings as well where your attendance is encouraged.
```
Comment (zhubx007) [inline reference/tsc-guide.md]: Its -> It's ?

**Review by FrankGTY — APPROVED** [review]

**Review by likai99c — APPROVED** [review]

File: reference/chair.md (hunk 2 above)
Comment (ghost) [reply reference/chair.md]: remove it done

File: reference/tsc-guide.md (hunk 3 above)
Comment (ghost) [reply reference/tsc-guide.md]: done

**Review by zhubx007 — APPROVED** [review]
LGTM

**Review by moweiwei — APPROVED** [review]
lgtm

File: reference/chair.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- Manage formal TSC meetings
+  - maintain the agenda in the online-document platform
+- Review proposals to the openv2x/governance repo
+- Coordinate the agenda for joint leadership meetings with the board
+- Send weekly update emails to openv2x-discuss
+
+## Each Cycle
+
+- Manage the initiative tracker
+  - Prepare release tracker eventpad (During PTG), Example:
```
Comment (li-qi-cool) [inline reference/chair.md]: Any more after the example?

Comment (ghost) [reply reference/chair.md]: done

**Review by li-qi-cool — APPROVED** [review]

**Pull request merged**